modules/mobile/app/controllers/mobile/v0/payment_history_controller.rb

```diff
@@ -43,8 +42,26 @@ def bgs_service_response
       end
 
       def paginate(payments, validated_params)
+        payment_years = Set[]
```

Comment:
I'm not sure I understand the purpose of initializing as a set only to convert to an array. Why not just start with an array?

Reply:
I see. Outside of our codebase, I'm not used to seeing set a lot. Thanks for explaining.

Reply:
We ended up changing it to `uniq` anyway because we now need to know the dates before we filter so we don't have that loop.